### PR TITLE
[WIP] feat(scripts): Add Dockerfile for arm64

### PR DESCRIPTION
**Pull request by jhermsmeier**
This adds a Dockerfile to build Etcher on arm64

Change-Type: patch

### CODE REVIEW

File: scripts/build/docker/compile-template.js [hunk 1]
```diff
@@ -40,6 +40,10 @@ _.each([
   {
     architecture: 'armv7hf',
     image: 'resin/armv7hf-debian:jessie'
+  },
+  {
+    architecture: 'armv7hf',
```
Comment (lurch) [inline scripts/build/docker/compile-template.js]: Umm, 2 separate sections both with `architecture: 'armv7hf'` ??

File: scripts/build/docker/Dockerfile.template [hunk 2]
```diff
@@ -47,7 +50,7 @@ RUN apt-get update \
 RUN curl -sL https://deb.nodesource.com/setup_6.x | bash - \
   && apt-get install -y nodejs
 
-<% if (architecture != 'armv7hf') { %>
+<% if (architecture != 'armv7hf' && architecture != 'arm64' ) { %>
```
[the remark this answers is not in the document]
Comment (jhermsmeier) [reply scripts/build/docker/Dockerfile.template]: Yeah, that's better – changed!

File: scripts/build/docker/compile-template.js (hunk 1 above)
Comment (jhermsmeier) [reply scripts/build/docker/compile-template.js]: Ah yes, indeed – fixed!

**Comment by lurch** [comment]
If you add the below patch, then when we add more conditionals in future it should stop all the other Dockerfiles changing at the same time too :-)
```patch
--- a/scripts/build/docker/compile-template.js
+++ b/scripts/build/docker/compile-template.js
@@ -42,7 +42,7 @@ _.each([
     image: 'resin/armv7hf-debian:jessie'
   }
 ], (options) => {
-  const result = _.template(template)(options)
+  const result = _.replace(_.template(template)(options), /(\n\n)\n+/g, '$1')
   const filename = path.join(currentDirectory, `Dockerfile-${options.architectu
   fs.writeFileSync(filename, result)
 })
```

**Comment by thundron** [comment]
Better integrate this on concourse. Closing

**Pull request closed without merging**